workers/worker_base.py

```diff
@@ -420,30 +420,42 @@ def connect_to_broker(self):
         if not connected:
             sys.exit('Could not connect to the broker after 5 attempts! Quitting...\n')
 
-    def find_id_from_login(self, login):
-        """ Retrieves our contributor table primary key value for the contributor with
```

Comment:
Please make sure your PR includes the new method header for `find_id_from_login` which is now present in dev.

Reply:
Yeah will do this.